tests/e2e/specs/subscriptions/shopper/shopper-myaccount-renew-subscription.spec.js

```diff
@@ -91,6 +77,8 @@ describeif( RUN_SUBSCRIPTIONS_TESTS )(
 			await page.waitForNavigation( { waitUntil: 'networkidle0' } );
 			await shopper.placeOrder();
 			await expect( page ).toMatch( 'Order received' );
+
+			await shopper.logout();
```

Comment:
This can also be moved to an `afterEach` block for making it consistent if we add more tests to this file later?

Reply:
This can be done in this particular test. We only check shopper flows. But I believe we should explicitly logout and not do it inside teardown unless it's repetitive.

See https://github.com/Automattic/woocommerce-payments/pull/4657#discussion_r959863817

Reply:
Updated